spec/models/topic_spec.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id          :bigint           not null, primary key
+#  description :string(50)       not null
+#  image       :string
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+require 'rails_helper'
+
+RSpec.describe Topic, type: :model do
+	context 'Validations' do
+	  let!(:topics) { create_list(:topic, 15) }
```

Comment:
You don't need to create 15 topics.
And when testing validations there is no need to create a topic, you can build one. Creation needs a DB and is more expensive. With few tests, it doesn't matter, but when the project grows we want tests to be as fast as possible